### DISCUSSION TITLE
How to import the JSBSimFlightDynamicsModel into another UE 5 Project

### DISCUSSION THREAD

**Discussion by Thauume** [JSBSim-Team/jsbsim#669] [Support requests]
Hello, I have a question, i want to merge the JSBSim Plugins into another project that doesn't have the plugins in 

but when i build JSBSIMForUnreal.sln and build UEReferenceMap I try to take the plugins Folder and put it in my another project but that doesnt work and tell me he cannot build 

Thanks for your help

**Comment by gallonmate** [comment]
In the Readme there was a link to tutorial(below) with more details there. As with any compiled plugin for Unreal, you have to start by making your project c++. You can turn a blueprint project to c++ by going to Tools and add a blank c++ class. https://dev.epicgames.com/community/learning/tutorials/mmL/a-diy-flight-simulator-tutorial#initial-project-setup-6

**Reply by Thauume** [reply]
I did manage to import the JSBSim into another project but everytime i launch the game i get a crash :

LoginId:b12ceb0e48854ca70433efb4b49aca34
EpicAccountId:9e8f09ea16d540bebea510a23bddd0ca

Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x00000000000003d0

UnrealEditor_GeoReferencing
UnrealEditor_JSBSimFlightDynamicsModel!UJSBSimMovementComponent::PrepareJSBSim() [D:\jsbsim-2a5985c2e190ca05f5ecd55b480eed40216dedb6\UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\JSBSimFlightDynamicsModel\Private\JSBSimMovementComponent.cpp:372]
UnrealEditor_JSBSimFlightDynamicsModel!UJSBSimMovementComponent::BeginPlay() [D:\jsbsim-2a5985c2e190ca05f5ecd55b480eed40216dedb6\UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\JSBSimFlightDynamicsModel\Private\JSBSimMovementComponent.cpp:280]
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll

**Reply by Thauume** [reply]
The plugin works but if i change for example The structural Frame Origin or hit play

**Reply by Thauume** [reply]
the project crash

**Reply by Thauume** [reply]
(I just Copy and Paste the Plugins folder of UEReferenceApp Into mine
